source/Outfit.cpp

```diff
@@ -38,7 +39,14 @@ void Outfit::Load(const DataNode &node)
 	for(const DataNode &child : node)
 	{
 		if(child.Token(0) == "category" && child.Size() >= 2)
+		{
 			category = child.Token(1);
+			const vector<string> &outfitCategoryList = GameData::Categories("outfit");
+			const vector<string> &shipCategoryList = GameData::Categories("ship");
+			if(find(outfitCategoryList.begin(), outfitCategoryList.end(), category) == outfitCategoryList.end()
```

Comment:
You shouldn't do this check here (in Outfit::Load) as the state of GameData::Categories() is unknown while GameData::BeginLoad is still operating.

In GameData::FinishLoading, after you insert the Fighter/Drone categories, you know your category lists are complete. Given that GameData::CheckReferences is the major "hey is anything wonky?" call, in that method you should extend the check of outfits and ships for proper categories